mlir/tools/CMakeLists.txt

```diff
@@ -1,16 +1,16 @@
 #add_subdirectory(mlir-cuda-runner)
 #add_subdirectory(mlir-cpu-runner)
-add_subdirectory(miopen-lsp-server)
-add_subdirectory(miopen-opt)
-add_subdirectory(miopen-gen)
+add_subdirectory(rock-lsp-server)
+add_subdirectory(rock-opt)
+add_subdirectory(rock-gen)
```

Comment:
However, we could go either way on what to call this one. I think `rock-gen` makes sense, actually